plugins/cdi/cdi.plugin.zsh

```diff
@@ -0,0 +1,20 @@
+# vim: set sw=4 expandtab:
+#
+# Licence: GPL
```

Comment:
If you want to add this plugin to OMZ you're accepting adding it with the MIT license. I'm afraid these are both incompatible.